pages/download.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+import { get } from '@vueuse/core';
+import { commonAttrs, noIndex } from '~/utils/metadata';
+import { useAppDownload } from '~/composables/download';
+import { type DownloadItemProps } from '~/components/download/DownloadItem.vue';
+
+useHead({
+  title: 'download',
+  meta: [
+    {
+      key: 'description',
+      name: 'description',
+      content: 'Download rotki',
+    },
+    noIndex(),
```

Comment:
This we should probably remove since we want the page to be indexed.

Reply:
got it, will do.